updated release notes summary (#2438)
MicrosoftDocs/edge-developer: microsoft-edge/webview2/release-notes.md

Generally, release notes apply across the supported platforms, which are listed in [WebView2 API Reference](webview2-api-reference.md).

WebView2 changes may require an update to the Runtime, SDK, or both. Most new APIs require both Runtime and SDK updates. Starting with the February 2023 release, the update requirement for each bug fix is indicated as follows:

| Indicator | Meaning |
|---|---|
| No label | Both the Runtime and the SDK need to be updated. |
| **Runtime-only** | Only the Runtime needs to be updated. |
| **SDK-only** | Only the SDK needs to be updated. |

> WebView2 shares code and binaries with the Microsoft Edge browser, and is released around the same time. As a result, WebView2 Runtime releases generally also include Microsoft Edge updates. For Microsoft Edge updates, see [Stable Channel Release Notes](/deployedge/microsoft-edge-relnote-stable-channel) and [Beta Channel Release Notes](/deployedge/microsoft-edge-relnote-beta-channel).

Reviewer: ```suggestion
WebView2 shares code and binaries with the Microsoft Edge browser, and is released around the same time. As a result, WebView2 Runtime releases generally also include Microsoft Edge updates. For Microsoft Edge updates, see [Release notes for Microsoft Edge Stable Channel](/deployedge/microsoft-edge-relnote-stable-channel) and [Release notes for Microsoft Edge Beta Channel](/deployedge/microsoft-edge-relnote-beta-channel).
```
per corp style guide and product style guide & policies, where feasible, this repo sets the link text to the literal title or heading of the destination doc, to prevent disorientation.  might be able to provide a style guide link about this

Generally, release notes apply across the supported platforms, which are listed in [WebView2 API Reference](webview2-api-reference.md).

> WebView2 changes may require an update to the Runtime, SDK, or both. Most new APIs require both Runtime and SDK updates. Starting with our release in February 2023, all bug fixes will be marked as **Runtime-only** meaning only the Runtime needs to be updated, **SDK-only** meaning only the SDK needs to be updated, and no label means both need to be updated.

Reviewer: WebView2 changes may require an update to the Runtime, SDK, or both. Most new APIs require both Runtime and SDK updates. Starting with the February 2023 release, the update requirements for each bug fix will be indicated as follows:

| Indicator | Meaning |
|---|---|
| No label | Both the Runtime and the SDK need to be updated. |
| **Runtime-only** | Only the Runtime needs to be updated. |
| **SDK-only** | Only the SDK needs to be updated. |

> WebView2 changes may require an update to the Runtime, SDK, or both. Most new APIs require both Runtime and SDK updates. Starting with our release in February 2023, all bug fixes will be marked as **Runtime-only** meaning only the Runtime needs to be updated, **SDK-only** meaning only the SDK needs to be updated, and no label means both need to be updated.

Reviewer: ```suggestion
WebView2 changes may require an update to the Runtime, SDK, or both. Most new APIs require both Runtime and SDK updates. Starting with the February 2023 release, the update requirements for each bug fix will be indicated as follows:

| Indicator | Meaning |
|---|---|
| No label | Both the Runtime and the SDK need to be updated. |
| **Runtime-only** | Only the Runtime needs to be updated. |
| **SDK-only** | Only the SDK needs to be updated. |
```

> WebView2 shares code and binaries with the Microsoft Edge browser and is released around the same time. As a result WebView2 Runtime releases generally also include Microsoft Edge updates. See [Stable Channel Release Notes](https://learn.microsoft.com/en-us/deployedge/microsoft-edge-relnote-stable-channel) and [Beta Channel Release Notes](https://learn.microsoft.com/en-us/deployedge/microsoft-edge-relnote-beta-channel) for Microsoft Edge updates.

Reviewer: ```suggestion
WebView2 shares code and binaries with the Microsoft Edge browser, and is released around the same time. As a result, WebView2 Runtime releases generally also include Microsoft Edge updates. For Microsoft Edge updates, see [Stable Channel Release Notes](/deployedge/microsoft-edge-relnote-stable-channel) and [Beta Channel Release Notes](/deployedge/microsoft-edge-relnote-beta-channel).
```
